svthalia/concrexit
# Disable Django sites module since it is obsolete

Closes #967

### Summary
This disable the Django sites module, which causes every instance of the codebase redirecting to Thalia.nu for some links.

### How to test
Steps to test the changes you made:
1. Go to the environment
2. Click on 'View on site' for an event you edit
3. You should be redirected to the environment you are using, not thalia.nu
